exonum/src/encoding/mod.rs

```diff
@@ -12,23 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! `encoding` is a lazy serialization library,
-//! it allows to keep struct serialized in place, and deserialize fields on demand.
+//! `encoding` is a lazy serialization library supporting zero-copy (de)serialization
```

Comment:
What meaning does the word "lazy" has here? I think this is **eager** serialization library, in that all messages are validated immediately, so deserialization is O(N) (in contrast to cap'n'protos O(1) deserialization).

Perhaps replace lazy with zero-copy, or just drop it?